packages/watcher/src/types.ts

```diff
@@ -0,0 +1,98 @@
+import type {
+  MediaListReference,
+  AddressReference,
+  MessengerReference
+} from '@windingtree/org.json-schema/types/org.json';
+import {
+  allowedSpaceTypes,
+  allowedLodgingFacilityTypes,
+  allowedLodgingFacilityTiers
+} from './enum';
+
+export type SpaceType =
+	typeof allowedSpaceTypes[number];
+
+export type LodgingFacilityType =
+	typeof allowedLodgingFacilityTypes[number];
+
+export type LodgingFacilityTier =
+	typeof allowedLodgingFacilityTiers[number];
+
+export type Amenity = string;
+
+export interface FacilityContactReference {
+  name: string;
+  phone: string;
+  email: string;
+  website: string;
+  messengers?: MessengerReference[];
+}
+
+export interface SpaceRaw {
```

Comment:
Always better to import already defined types. You can import these types from `data-models` package:

```
import type {
  SpaceRaw, // Original space data type
  LodgingFacilityRaw, // Original lodging facility data type
  Space, // Space storage record typescript type
  LodgingFacility // Lodging facility record typescript type
} from 'stays-data-models';
```